tests/drivers/gpio/gpio_basic_api/boards/frdm_ke17z512.overlay

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright NXP 2024
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	resources {
+		compatible = "test-gpio-basic-api";
+		out-gpios = <&gpioe 0 0>; /* Arduino D13 */
+		in-gpios  = <&gpioe 1 0>; /* Arduino D12 */
```

Comment:
you could define an arduino nexus node I believe in the board definition to make this type of thing more straightforward